routesrv/polling.go

```diff
@@ -79,6 +81,16 @@ func (p *poller) poll(wg *sync.WaitGroup) {
 		if p.oauth2Config != nil {
 			routes = p.oauth2Config.NewGrantPreprocessor().Do(routes)
 		}
+		if p.editRoute != nil {
+			for _, editor := range p.editRoute {
+				routes = editor.Do(routes)
+			}
+		}
+		if p.cloneRoute != nil {
+			for _, cloner := range p.cloneRoute {
+				routes = cloner.Do(routes)
+			}
+		}
```

Comment:
Route processing starts to grow, lets have a helper `process` or `prepare` method  and move all preprocessors as well as `sort.SliceStable` (from below) there.